pkg/controller/migmigration/stage.go

```diff
@@ -562,6 +562,16 @@ func (t *Task) ensureStagePodsTerminated() (bool, error) {
 	return true, nil
 }
 
+// Label applied to all stage pods for easy cleanup
+func (t *Task) stagePodIdentifierLabel() map[string]string {
+	return map[string]string{StagePodLabel: migapi.True}
+}
+
+// Build map of all stage pod labels
```

Comment:
I decided to leave the correlation labels in place, they could be useful for something I think and no harm in leaving them on a resource we will delete soon anyways.